pick.js

```diff
@@ -16,20 +16,27 @@
  * console.log(
  *   pick(['hello', 'world'])({ goodbye: 1, world: 2 }),
  * ) // { world: 2 }
+ * console.log(
+ *   pick(['a.b.c.d'])({ a: { b: { c: { d: 1, e: [2, 3] } } } }),
+ * ) // { a: { b: { c: { d: 1 } } } }
  * ```
  */
+
+const getByPath = require("./_internal/getByPath")
+const set = require("./set")
```

Comment:
this might be what is causing the build step to hang.. maybe. Let's use setByPath here instead of set. It is typically more performant to supply all arguments at once in a single function call as opposed to multiple calls to a higher order function.

Reply:
pick is now using setByPath, `node distributor.js` is still hanging